src/ngx_http_lua_kong_module.c

```diff
@@ -128,7 +136,22 @@ ngx_http_lua_kong_create_loc_conf(ngx_conf_t* cf)
         return NULL;
     }
 
+    conf->error_log_append = NGX_CONF_UNSET_PTR;
```

Comment:
Do we really need it? Is `NULL` not enough? I'm not sure about it.

Reply:
it works both ways but I think this is more consistent with other modules